tests/test_distributions.py

```diff
@@ -235,3 +235,27 @@ def test_repr():
 
     for d in EXAMPLE_DISTRIBUTIONS.values():
         assert d == eval('distributions.' + repr(d))
+
+
+def test_str_uniform_distribution():
```

Comment:
Please update test names.
```suggestion
def test_uniform_distribution_as_dict():
```

Reply:
I renamed to `_asdict`.